### PR TITLE
[TASK] composer normalize

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -48,6 +48,16 @@
         "a9f/fractor-yaml": "self.version",
         "a9f/typo3-fractor": "self.version"
     },
+    "replace": {
```
Comment (andreaswolf) [inline composer.json:51]: @simonschaufi why did you add exactly the same "replace" key again?

Comment (simonschaufi) [reply composer.json:51]: that was not intentional but happened because of the rebase. I'll fix this.
